persistence/src/main/resources/db/initHSQLDB.sql

```diff
@@ -9,4 +10,13 @@ CREATE TABLE users (
 
 CREATE UNIQUE INDEX login ON users (login);
 
-CREATE UNIQUE INDEX email ON users (email);
\ No newline at end of file
+CREATE UNIQUE INDEX email ON users (email);
+
+CREATE TABLE posts (
```

Comment:
@slav9nin I think it would be better to use some fixture written on Java (it will save us from vendor lock)